airflow/www/static/js/task.js

```diff
@@ -25,12 +25,17 @@ import validator from "validator";
 document.addEventListener("DOMContentLoaded", () => {
   document.querySelectorAll(".js-ti-attr").forEach((attr) => {
     const value = attr.innerHTML;
+    const attrName = attr.previousElementSibling.textContent.trim();
```

Comment:
Let me know if this is alright or if there's a better solution than this. I focused on solving the issue as described in #28482, and probably overlooked a lot of things.